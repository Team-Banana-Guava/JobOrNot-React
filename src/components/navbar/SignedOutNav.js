import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../../assets/JobOrNot_Blue_50.png';

const SignedOutNav = (props) => {
    return (
        <nav className='navbar'>
            <div className='container'>
                <div className='row'>
                    <div className='three columns'>
                        <Link to='/'><img src={logo} alt='Job or Not' /></Link>
                    </div>
                    <div className='nine columns'>
                        <Link to='/signin' className='u-pull-right'>Sign In</Link>
                        <Link to='/howitworks' className='u-pull-right'>How It Works</Link>
                    </div>
                  </div>
              </div>
        </nav>
    );
};


export default SignedOutNav;
