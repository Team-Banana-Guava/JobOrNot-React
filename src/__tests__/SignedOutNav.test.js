/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SignedOutNav from '../components/navbar/SignedOutNav';

describe('SignedOutNav', function () {
    it('renders the SignedOutNav component', () => {
        const rendered = shallow(<SignedOutNav button={<button>button node</button>} />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});
