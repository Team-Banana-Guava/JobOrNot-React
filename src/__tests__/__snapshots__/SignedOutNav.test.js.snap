exports[`SignedOutNav renders the SignedOutNav component 1`] = `
<nav>
  <ul>
    <li>
      <img
        alt="logo"
        src="http://placehold.it/50x50" />
    </li>
    <li>
      JobOrNot
    </li>
    <li>
      How It Works
    </li>
    <li>
      <button>
        button node
      </button>
    </li>
  </ul>
</nav>
`;
